src/index.ts

```diff
@@ -1,25 +1,475 @@
 import { install } from "source-map-support";
 import { config } from "dotenv";
 import Octokit from "@octokit/rest";
-import logger from "./lib/logger";
-import client from "./lib/client";
+import logger from "./logger";
+import client from "./client";
+import userConfig from "./config";
+import { isEmpty, size, includes } from 'lodash';
 
 // Setup env variables and source maps
-config();
 install();
 
 logger.info("Application has been initialized.");
 logger.info("Running in mode %s", process.env.NODE_ENV);
 
-async function getIssues(client: Octokit) {
-  const issues = await client.issues.listForRepo({
-    owner: "cwramsey",
-    repo: "issue-testing"
+let allLabelsList: Array<any> = []
+
+async function getIssues(client: Octokit, pageNum: number, maxPerPage: number) {
+
+  let allIssues: Array<any> = []
+  while (true) {
+    const { data, status } = await client.issues.listForRepo({
+      owner: userConfig.owner,
+      repo: userConfig.repo,
+      per_page: maxPerPage,
+      page: pageNum
+    });
+
+    if (status != 200) {
+      const error = new Error(`Failed to fetch issues`)
+      logger.info("error:", status)
```

Comment:
You can use `logger.error` instead of info to prefix log lines accordingly

Reply:
changed 3 places